lib/dry/files.rb

```diff
@@ -736,6 +752,11 @@ def remove_block(path, target)
 
     private
 
+    # @since 0.2.0
+    # @api private
```

Comment:
@alassek If this is passed as default argument to `#open`, which is a public method, by consequence, this is public API as well.
```suggestion
    # @api public
```